pkg/server/chaosd/server.go

```diff
@@ -32,6 +33,7 @@ type Server struct {
 func NewServer(
 	conf *config.Config,
 	exp core.ExperimentStore,
+	expRun core.ExperimentRunStore,
```

Comment:
Can we manage the experiment's status in `core.ExperimentStore`? And don't need to add a new interface

Reply:
Yep that can be done, will switch to that